warrior/Framework/ClassUtils/warrior_connect_class.py

```diff
@@ -415,7 +416,10 @@ def connect_ssh(self):
         if WarriorCliClass.cmdprint:
             pNote("connectSSH: :CMD: %s" % command)
             return None, ""
-        child = self.pexpect.spawn(command, timeout=int(self.timeout))
```

Comment:
lines 419-422 are repeated in several places in this PR, is it possible to convert it to a generic function in cli_utils